apps/api/internal/api/v1/moderation/service.go

```diff
@@ -104,3 +108,55 @@ func handleUpdate(
 	}
 	return settings, nil
 }
+
+func handlePostTitle(channelId string, dto *postTitleDto, services types.Services) (*postTitleResponse, error) {
+	logger := do.MustInvoke[interfaces.Logger](di.Provider)
+	tokensGrpc := do.MustInvoke[tokens.TokensClient](di.Provider)
+	config := do.MustInvoke[cfg.Config](di.Provider)
+
+	twitchClient, err := twitch.NewUserClient(channelId, config, tokensGrpc)
+	if err != nil {
+		logger.Error(err)
+		return nil, fiber.NewError(http.StatusInternalServerError, "internal error")
+	}
+
+	resp, err := twitchClient.EditChannelInformation(&helix.EditChannelInformationParams{
+		BroadcasterID: channelId,
+		Title:         dto.Title,
+	})
+	if err != nil {
```

Comment:
err is golang error, not http. So that lib returns `resp.StatusCode` and `resp.ErrorMessage` which is string. You should cover that.

Reply:
Fixed.